pkgs/development/python-modules/pymumble/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage,
+  fetchFromGitHub,
+  isPy27,
+  opuslib,
+  protobuf,
+  stdenv,
```

Comment:
since we're just uring stdenv to access lib, we could just access lib directly
```suggestion
  lib,
```

Reply:
thanks for the advice, never realized its the same `lib`

Reply:
it's there just for convenience